docs/esg.cordex-cmip6.ini

```diff
@@ -0,0 +1,124 @@
+[project:cordex-cmip6]
+
+# Define the categories to be used for this project:
+#   name | category_type | is_mandatory | is_thredds_property | display_order
+categories =
+    mip_era               | enum   | true  | true  | 0
+    activity_id           | enum   | true  | true  | 1
+    domain_id             | enum   | true  | true  | 2
+    driving_source_id     | enum   | true  | true  | 3
+    driving_experiment_id | enum   | true  | true  | 4
+    driving_variant_label | string | false | true  | 5
+    institution_id        | enum   | true  | true  | 6
+    source_id             | enum   | true  | true  | 7
+    version_realization   | enum   | true  | true  | 8
+    variable_id           | string | true  | true  | 9
+    frequency             | string | false | true  | 10
+    product               | string | false | true  | 11
+    source_type           | string | false | true  | 12
+    grid                  | string | false | true  | 13
```

Comment:
Not sure if `grid` and other CMIP6 DRS elements that in CORDEX are just global attributes should remain here as category